### PR TITLE
Edit entries c sharp add codebyte 1

### PR DESCRIPTION
**Pull request by cslylla**
### Description

- The following terms were updated:
  - C# Math functions: .Ceiling(),
  - C# Math functions: .Exp(),
  - C# Math functions: .Floor().
- The following changes were made:
  - Add codebyte example,
  - Fix typos.
- Closes: #1799 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/math-functions/terms/floor/floor.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ CatalogContent:
...
+**`Math.Floor()`** is a static method that rounds down and returns the largest whole integer that is less than or equal to the given number.
 
 ## Syntax
 
 ```pseudo
 Math.Floor(x);
 ```
 
-- The `Math.Floor()` method is used to find the largest whole integer which is less than or equal to the argument, `x`, that is passed in. The argument can be of a float or double type.
+- The `Math.Floor()` method is used to find the largest whole integer that is less than or equal to the argument, x, passed in. The argument can be of a `float` or `double` type.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/math-functions/terms/floor/floor.md:24]:
```suggestion
- The `Math.Floor()` method is used to find the largest whole integer that is less than or equal to the argument, `x`, passed in. The argument can be of a `float` or `double` type.
```

**Review by SSwiniarski — APPROVED** [review]
@cslylla Looks good for a second review.

**Review by KTom101 — APPROVED** [review]
@cslylla This looks good as is! 👍

**Comment by KTom101** [comment]
@cslylla Edits are live: [.Ceiling()](https://www.codecademy.com/resources/docs/c-sharp/math-functions/ceiling), [.Exp()](https://www.codecademy.com/resources/docs/c-sharp/math-functions/floor), [.Floor()](https://www.codecademy.com/resources/docs/c-sharp/math-functions/floor)

Screenshot 2023-03-30 at 10 28 22 PM

Screenshot 2023-03-30 at 10 28 38 PM

Screenshot 2023-03-30 at 10 29 17 PM

**Pull request merged**